.github/labels.md

```diff
@@ -0,0 +1,252 @@
+state of Issues/PRs without explicitly reading through lengthy comments, code
+reviews or commits.  Please follow this guide as closely as possible to avoid
+duplicating work and wasting time of other developers and to enable better
+documentation on decisions made.
+
+
+## Label categories
+
+Labels are grouped in categories.  Those categories group together a common
+concept like the resolving state of an Issue/PR or the target platform where the
+Issue/PR occurs.  Categories share a prefix in the form
+`category_name:label_name` and are grouped together by color.  As a rule of
+thumb there is at least one label from every category that needs to be applied
+to an issue but use common sense to check if the label is appropriate or not for
+the specific Issue/PR.
+
+Currently there are three categories:
+
+
+#### Label category `category`
+
+Labels within this category describe the intent of the Issue/PR.  Usually there
+should be exact one intent for creating an Issue/PR and it should not change
+during the lifetime of an Issue/PR.
+
+#### Label category `component`
+
+The component lists the parts of the project this Issue/PR affects or is
+affected by.  This should help developers to identify if they have required
+knowledge or interests to handle the issue.  There should be at least one
+component listed for an Issue.
+
+#### Label category `status`
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+Issue/PR have exactly one status.  Status may change over the lifetime of the
+Issue/PR.
+
+
+## Labels
+
+This section lists when an label should be applied and what additional actions
+may be taken.
+
+
+#### Label `category:bug`
+
+The Issue/PR describes or solves a perceived malfunction within the game.
+
+#### Label `category:feature`
+
+The Issue/PR describes or implements a new game feature.  Features are parts of
+the game which a player can interact with e.g.: A new species, a new gameplay
+mechanic or a new command line switch for the server that changes gameplay
+behavior.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Issue/PR describes or contains a new or improved implementation of an
+existing feature or any other code restructuring that doesn't modify the current
+game mechanics or representation.
+
+#### Label `category:request`
+
+??? Requests don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+The PR contains insignificant code changes that don't involve features,
+refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue/PR deals with the python AI decision making code or affects it.
```

Comment:
Ok, something I constantly run into and always forget to ask, so I'm using this case here now: question for our native speakers: shouldn't "python" be spelled "Python" here (uppercase)? Because it's a proper name, and those are spelled with capital first letter IIRC...